### ISSUE TITLE
Winter Break TODO

### ISSUE THREAD

**Issue by EricChen424** [bcgsc/GapPredict#44]
1) Hook up the GPU code and see how much faster it is
- right now we take about 50 minutes to handle 5 million 26-mers with the Vanilla DNN
- try to increase batch size
- can we do a conv net to do word embedding on our kmer, then shrink the size of our RNN hidden layer size because our embedded kmer simplifies things a bit?
* the input to our RNN is a matrix of kmers that all overlap by k-1 bases that have been embedded to vector form by a CNN trained on a random sample of kmers (we'd like to train on all possible kmers of some length k but this is exponential in length so not really)
https://machinelearningmastery.com/cnn-long-short-term-memory-networks/

2) We want graphs for 1 kb, 10 kb, 100 kb, 1 Mb
- for training time
- for training accuracy
- for validation accuracy
- benchmarks with respect to coverage (1X, 10X, 20X, 30X, etc... just take a random sample), k-mer length, model architecture (ie. hyperparameters)

https://arxiv.org/abs/1611.02683
Unsupervised training?
